Update var-study-stmt-source.yaml

mbrush opened the pull request:
- Proposes minor changes to descriptions of the 'study summary'  statement classes, to better align with language and context provided by the large gks-core and sepio models. 
     - Specifically, clarify that these classes report statements _scoped to report a conclusion from a single study_ - based on results/evidence from that study.  
     - This will be important to appreciate how these study-scoped are different than the broader statements making more definitive assertions about a variant's diagnostic/prognostic/predictive associations - which will eventually be made, and which will use the study-scoped statements as evidence.

- Makes minor updates to qualifier attribute descriptions, to make more informative and consistent

- Makes predicate enum labels consistent: some predicates read as relationships /verb phrases, while others do not.  Proposed updated names that frame all as verb phrases (framing predicates as relationships is standard practice in sepio, and SPO models more generally, and gives a clear, readable SPO sentence).

mbrush committed:
Update var-study-stmt-source.yaml

- Proposes minor changes to descriptions of the 'study summary'  statement classes, to better align with language and context provided by the large gks-core and sepio models. Specifically, clarify that these classes report statements _scoped to report a conclusion from a single study_ - based on results/evidence from that study.  This will be important to appreciate how these study-scoped are different than the broader statements making more definitive assertions about a variant's diagnostic/prognostic/predictive associations - which will eventually be made, and which will use the study-scoped statements as evidence.

- Minor updates to qualifier attribute descriptions to make more specific and consistent

- Made predicate enum labels consistent: some predicates read as relationships /verb phrases, while others do not.  Proposed updated names that frame all as verb phrases (framing predicates as relationships is standard practice in sepio, and SPO models more generally, and gives a clear, readable SPO sentence).

mbrush committed:
Update var-study-stmt-source.yaml

minor capitalization fixes

larrybabb commented on schema/profiles/var-study-stmt-source.yaml:
```diff
@@ -35,8 +36,8 @@ $defs:
         enum:
-        - positive
-        - negative
+        - supports_positive_diagnosis_for
```
I'm going to defer to @ahwagner on these predicate term changes.  I do like the idea of making them consistent as you mentioned in your description of this PR. I do think we should stick to using camelCase instead of snake_case since we've started with camelCase in the other profiles (unless you've changed those as well). 
I'll defer approving this until @ahwagner weighs in since these profiles are in his territory.

larrybabb replied:
@ahwagner can you weigh in on this comment so we can modify and approve or simply approve as-is?

ahwagner replied:
agree with case conventions.

Need to give a little thought to the predicate conventions here. "Positive" means that the presence of the variant is a positive indicator for a disease diagnosis (inclusion criteria for a disease); "negative" means that the presence of the variant is negative indicator for a disease diagnosis (exclusion criteria for). Not sure I like the use of "negative diagnosis" and "positive diagnosis", though; that seems more like a test result than inclusion / exclusion criteria.

mbrush replied:
Happy to come up with a different phrase for the predicate.  As long as it is a verb phrase that expresses the relationship between the variant and the disease in this statement types, and not just the word "positive".  

@ahwagner based on your clarifications would one of the the following work better:

- `positiveIndicatorFor`
- `positiveIndicatorForDisease`
- `positiveDiagnosticIndicatorFor`
- `positiveIndicatorForDiagnosisOf`

. . . or perhaps you have better suggestions?

ahwagner replied:
perhaps `positive` -> `isDiagnosticInclusionCriterionFor` and `negative` -> `isDiagnosticExclusionCriterionFor`?

mbrush replied:
I will go with whatever verb phrase you think is best here Alex.  I'll update the PR to use your suggestion above for now.  We can always update the predicate labels later.

Note that to reflect this change in the enums to reference inclusion/exclusion criteria instead of positive/negative diagnoses, I updated the text description of this Statement type to use similar language.  

Finally, note that I also updated the predicate enums for the Prognostic statement type to be verb phrases (e.g. `Better Outcome` --> `associatedWithBetterOutcomeFor`).  Just mirrored the language in your text definition here, but happy to change.

mbrush committed:
Update var-study-stmt-source.yaml

Updating predicate enum labels for diagnostic and prognostic statement types

mbrush committed:
Update var-study-stmt-source.yaml

updated text description of diagnostic Statement type to match updated predicate enum terms.

ahwagner reviewed (approved):
LGTM. This is a solid consistency improvement, thanks @mbrush.

larrybabb committed:
Merge branch '1.x' into mbrush-patch-1
